helm/alfresco-content-services/templates/config-share.yaml

```diff
@@ -10,7 +10,6 @@ data:
   {{ $alfprotocol := tpl (.Values.externalProtocol | default "http") $ }}
   {{ $alfport := tpl (.Values.externalPort | default .Values.repository.service.externalPort | toString) $ }}
   {{ $alfproxy := printf "-Dalfresco.proxy=%s://%s:%s" $alfprotocol $alfhost $alfport }}
-  CATALINA_OPTS: {{ index .Values "share" "environment" "CATALINA_OPTS" }}
```

Comment:
This doesn't look correct, there are valid JVM options specified in the values file so not sure why this has been removed.

Reply:
if you generate the template it's duplicated as the environment dict is dumped after, I validated the output with the k8s schema

Reply:
Ah, gotcha, nice catch in the case!